tests/README.md

```diff
@@ -16,7 +16,27 @@ CONNECTION_PARAMETERS = {
     'warehouse': '<warehouse>',
 }
 ```
-Note that this file will be ignored by git, so you do not need to worry about check in your secret.
+Snowpark pandas API also offers the convenience of implicit session creation from a configuration file.
```

Comment:
Implicit session creation is useful to all users, not just to Snowpark pandas users. Can we describe it as such? We can say "this feature is particularly useful to Snowpark pandas users, because you can write your Snowpark pandas code almost as you would normally write pandas code."

Reply:
Good point, rephrased.